cmd/snap/main.go

```diff
@@ -359,7 +359,7 @@ func run() error {
 			return err
 		}
 
-		fmt.Fprintf(Stderr, msg)
+		fmt.Fprintf(Stderr, "%s\n", msg)
```

Comment:
`fmt.Fprintln()` will avoid the formatting overhead

Reply:
not sure overhead is the problem here,  anyway we use both styles around afaict, I changed this and another obvious one, to switch more places we need a dedicated PR though